public/js/activePlayersBootstrap.js

```diff
@@ -72,14 +89,15 @@ var ActivePlayersViewModel = function () {
   };
 
   this.ConvertToObservablePlayer = function (player) {
+    CacheCardDiv('game', player.GameID, player.GameTitle, player.ConsoleName, player.GameIcon);
```

Comment:
I'm not that familiar with `ko`. Is this going to be problematic?

Reply:
I don't think so. I'm not sure why those are observables looking at it.

Reply:
Seems to work just fine for knockout (`ko`).